### ISSUE TITLE
Simulate Retro TV doesn't work...

### ISSUE THREAD

**Issue by java-py-c-cpp-js** [Dariasteam/Cows-Revenge#12]
...neither on Android (v. 1.0.5 F-Droid) nor on Linux (v. 1.3 Flatpak). On both systems, a white cover covers the whole game:
![white](https://user-images.githubusercontent.com/26502842/45179135-e7568880-b217-11e8-83b4-67fd12e4c2a3.jpg)

**Comment by Dariasteam** [comment]
Yep, this thing its been a headache for a time. The shader used is not created by me, but [herique lalves](https://github.com/henriquelalves/SimpleGodotCRTShader). And it appear to work randomly in base of godot version or something. The current status is it works if playing the game inside godot editor but not when exported so I guess I should learn about shaders and check that.

If someone have an idea about this I'm all ears.

**Comment by java-py-c-cpp-js** [comment]
Currently I don't have time to test that, but I had a similar experience with Godot behaving different after exporting a (very simple) game. The error occured because the IDE was updated and I don't update the export templates, too, so they mismatch.

**Comment by Dariasteam** [comment]
I don't think that could be the problem in this case, because for the fdroid and flatpak versions both godot and export templates are compiled when building.
